### PR TITLE
Markdown conversion for `ru` - Replace - Media section :warning: Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of: #8563
Conversion report: https://gist.github.com/SphinxKnight/e66a41050cc8314ab9fcfd527ef8201f#file-md-conversion-problems-report-2022-09-29t11_25_07-036z-md

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/ru/web/media/formats/codecs_parameter/index.md [hunk 1]
```diff
@@ -0,0 +1,786 @@
...
+      <td>
+        Двухзначный номер уровня, который преобразуется в формат X.Y, где<code
+          >X = 2 + (LL >> 2)</code
+        >
+        , и <code>Y = LL &#x26; 3</code>. Подробнее
+        <a href="https://aomediacodec.github.io/av1-spec/#levels"
+          >Дополнение A, секция 3</a
+        >
+        в спецификации AV1 .
+      </td>
```
Comment (sashasushko) [inline files/ru/web/media/formats/codecs_parameter/index.md:126]:
```suggestion
      <td>
        Двухзначный номер уровня, который преобразуется в формат X.Y, где <code>X = 2 + (LL >> 2)</code>,
        и <code>Y = LL &#x26; 3</code>. Подробнее
        <a href="https://aomediacodec.github.io/av1-spec/#levels">Дополнение A, секция 3</a>
        в спецификации AV1 .
      </td>
```

File: files/ru/web/media/formats/codecs_parameter/index.md [hunk 2]
```diff
@@ -0,0 +1,786 @@
...
+    </tr>
+    <tr>
+      <td><code>T</code></td>
+      <td>
+        The one-character tier indicator. For the Main tier (<code
+          >seq_tier</code
+        >
```
Comment (sashasushko) [inline files/ru/web/media/formats/codecs_parameter/index.md:133]:
```suggestion
        The one-character tier indicator. For the Main tier (<code>seq_tier</code>
```

File: files/ru/web/media/formats/codecs_parameter/index.md [hunk 3]
```diff
@@ -0,0 +1,786 @@
...
+      <td><code>cp</code></td>
+      <td>
+        The two-digit <code>color_primaries</code> value indicates the color
+        system used by the media. For example, BT.2020/BT.2100 color, as used
+        for HDR video, is <code>09</code>. The information for this—and for each
+        of the remaining components—is found in the
+        <a
+          href="https://aomediacodec.github.io/av1-spec/#color-config-semantics"
+          >Color config semantics section</a
+        >
```
Comment (sashasushko) [inline files/ru/web/media/formats/codecs_parameter/index.md:229]:
```suggestion
        <a href="https://aomediacodec.github.io/av1-spec/#color-config-semantics">Color config semantics section</a>
```

File: files/ru/web/media/formats/codecs_parameter/index.md [hunk 4]
```diff
@@ -0,0 +1,786 @@
...
+    </tr>
+    <tr>
+      <td><code>cp</code></td>
+      <td>
+        <p>
+          A two-digit integer specifying which of the color primaries from
+          Section 8.1 of the
+          <a href="https://www.itu.int/rec/T-REC-H.273/en"
+            >ISO/IEC 23001-8:2016</a
+          >
```
Comment (sashasushko) [inline files/ru/web/media/formats/codecs_parameter/index.md:608]:
```suggestion
          <a href="https://www.itu.int/rec/T-REC-H.273/en">ISO/IEC 23001-8:2016</a>
```

**Pull request merged**